### PR TITLE
Some minor improvements

### PR DESCRIPTION
**Pull request by igormukhingmailcom**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -80,14 +80,69 @@
...
+        ],
+        "ensure-env-copied": "([[ ${SYMFONY_ENV:-'dev'} == 'dev' ]] && composer ensure-env-dev-copied) || ([[ ${SYMFONY_ENV:-'dev'} == 'test' ]] && composer ensure-env-test-copied) || echo 'Unknown environment ${SYMFONY_ENV}'",
+        "ensure-env-dev-copied": "(cd tests/Application && ([[ -f .env.dev.local ]] || cp .env  .env.dev.local))",
+        "ensure-env-test-copied": "(cd tests/Application && ([[ -f .env.test.local ]] || cp .env.test .env.test.local))",
+        "ensure-node-modules-installed": "[[ -d tests/Application/node_modules ]] || (cd tests/Application && yarn install)",
+        "ensure-schema-updated": [
+            "composer ensure-env-copied",
+            "(cd tests/Application && bin/console doctrine:schema:update --force -e ${SYMFONY_ENV:-'dev'})"
+        ],
+        "ensure-test-container-exists": "[[ -f tests/Application/var/cache/test/ApplicationTests_Setono_SyliusMiintoPlugin_Application_KernelTestDebugContainer.xml ]] || (cd tests/Application && php -d memory_limit=-1 bin/console cache:clear --env=test)",
```
Comment (loevgaard) [inline composer.json]: The test container isn't needed for PHPStan to work anymore.

File: README.md [hunk 2]
```diff
@@ -84,6 +85,19 @@ If you haven't added a GTIN field to your variants, you can use the [Barcode plu
 
 If you don't use GTIN's you should override the `ProductVariantMapper` service (`setono_sylius_miinto.mapper.product_variant`) with your own implementation.
 
+### Sandbox configuration
```
Comment (loevgaard) [inline README.md]:
```suggestion
#### Sandbox configuration
```

File: src/EventListener/StartProcessingSubscriber.php [hunk 3]
```diff
@@ -53,19 +54,36 @@ public function validate(GuardEvent $event): bool
         }
 
         if ($shop->getChannel() === null) {
-            return $this->block($event, \Safe\sprintf('The shop %s is not mapped to a channel', (string) $shop->getId()));
+            return $this->block($event, sprintf(
+                'The shop %s is not mapped to a channel.' .
```
Comment (loevgaard) [inline src/EventListener/StartProcessingSubscriber.php]:
```suggestion
                'The shop %s is not mapped to a channel. ' .
```

File: src/EventListener/StartProcessingSubscriber.php [hunk 4]
```diff
@@ -53,19 +54,36 @@ public function validate(GuardEvent $event): bool
...
+                'You should go to Admin > Miinto > Shops and choose channel for this shop.',
+                (string) $shop->getId()
+            ));
         }
 
         if ($shop->getLocale() === null) {
-            return $this->block($event, \Safe\sprintf('The shop %s is not mapped to a locale', (string) $shop->getId()));
+            return $this->block($event, sprintf(
+                'The shop %s is not mapped to a locale.' .
```
Comment (loevgaard) [inline src/EventListener/StartProcessingSubscriber.php]:
```suggestion
                'The shop %s is not mapped to a locale. ' .
```

File: src/EventListener/StartProcessingSubscriber.php [hunk 5]
```diff
@@ -53,19 +54,36 @@ public function validate(GuardEvent $event): bool
...
+                'You should go to Admin > Miinto > Shops and choose locale for this shop.',
+                (string) $shop->getId()
+            ));
         }
 
         if (!$this->shippingTypeMappingRepository->hasValidMapping($shop, $order->getShippingType())) {
-            return $this->block($event, \Safe\sprintf('The shipping type %s on shop %s is not mapped', $order->getShippingType(), (string) $shop->getId()));
+            return $this->block($event, sprintf(
+                'The shipping type %s on shop %s is not mapped.' .
```
Comment (loevgaard) [inline src/EventListener/StartProcessingSubscriber.php]:
```suggestion
                'The shipping type %s on shop %s is not mapped. ' .
```

File: src/EventListener/StartProcessingSubscriber.php [hunk 6]
```diff
@@ -53,19 +54,36 @@ public function validate(GuardEvent $event): bool
...
+                'You should go to Admin > Miinto > Shipping type mappings and choose shipping method for it.',
+                $order->getShippingType(),
+                (string) $shop->getId()
+            ));
         }
 
         if (!$this->paymentMethodMappingRepository->hasValidMapping($shop)) {
-            return $this->block($event, \Safe\sprintf('The shop %s is not mapped to a payment method', (string) $shop->getId()));
+            return $this->block($event, sprintf(
+                'The shop %s is not mapped to a payment method.' .
```
Comment (loevgaard) [inline src/EventListener/StartProcessingSubscriber.php]:
```suggestion
                'The shop %s is not mapped to a payment method. ' .
```

File: composer.json (hunk 1 above)
Comment (igormukhingmailcom) [reply composer.json]: Thanks

**Pull request merged**